src/client.ts

```diff
@@ -235,3 +240,25 @@ function updateTerminalSize(): void {
 }
 
 window.onresize = () => updateTerminalSize();
+
+const onDidChangeState = new Emitter<void>();
```

Comment:
in theory should not be necessary after https://github.com/gitpod-io/gitpod/pull/17124/commits/88fda89fedaa77a493fb7c35451f0489714372fb but does not harm as well and keep be used for lazily init later